remoteworkitem/github.go

```diff
@@ -35,7 +36,8 @@ func (f *githubIssueFetcher) listIssues(query string, opts *github.SearchOptions
 func (g *GithubTracker) Fetch() chan TrackerItemContent {
 	configuration, err := config.GetConfigurationData()
 	if err != nil {
-		panic(fmt.Errorf("Failed to setup the configuration: %s", err.Error()))
+		fmt.Print(errors.Wrap(err, "Failed to setup the configuration"))
```

Comment:
@kwk  Do you have further suggestions on this? 
I was wondering how the error is to be implemented . Returning `(chan TrackerItemContent, <-chan error)` is problematic because the invocation in scheduler.go refuses to accept this..

I was reading online and everywhere the suggestion was to use an error field inside the channel ( which is of type `TrackerItemContent` here ).

Do you think something like this would make sense?
```
tic := make(chan TrackerItemContent)
tic <- TrackerItemContent{err: errors.Wrap(err, "Failed to setup the configuration")}
close(tic)
return tic
```
here https://github.com/almighty/almighty-core/pull/712/files#diff-273ed465c4217f800eddd24f6261f282R36
Apologies if the above is absurd, I'm not great with `chan` atm :)